src/components/NewBulkUploadPanel.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { DocumentDuplicateIcon } from '@heroicons/react/24/outline';
+import { Panel, PanelBody } from './Panel';
+import './NewBulkUploadPanel.css';
+
+interface NewBulkUploadPanelProps {
+  apiUrl: URL;
+  bulkUploaderLoader: React.ReactNode;
+  baseFieldsLoader: React.ReactNode;
+}
+
+export const NewBulkUploadPanel = ({
+  apiUrl,
+  bulkUploaderLoader,
+  baseFieldsLoader,
+}: NewBulkUploadPanelProps) => {
+  const bulkUploadTemplateUrl = new URL('/static/bulkUpload.csv', apiUrl);
+
+  return (
+    <Panel>
+      <PanelBody>
+        <section id="bulk-upload-instructions">
+          <div className="instructions">
+            <h1>New bulk upload</h1>
```

Comment:
Just noticed: `Base Fields` header below uses Capital Case but this doesn't.  Probably worth being consistent.